aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,266 @@
+| Long written format            | {Intl.DateTimeFormat('en-US', { weekday: 'short', month: 'long', day: 'numeric', year: 'numeric' }).format(date)} | When space is available and displaying the day of week is beneficial to the user.                                                                                                                                                                                                      | <Box>`Intl.DateTimeFormat('en-US', { weekday: 'short',  day: 'numeric',month: 'long', year: 'numeric' }).format(date)`</Box> |
+| Medium written format          | {Intl.DateTimeFormat('en-US', { dateStyle: 'long' }).format(date)}                                                | When space is available.                                                                                                                                                                                                                                                               | <Box>`Intl.DateTimeFormat('en-US', { dateStyle: 'long' }).format(date)`</Box>                                                |
+| Short written format           | {Intl.DateTimeFormat('en-US', { dateStyle: 'medium' }).format(date)}                                              | When space is limited.                                                                                                                                                                                                                                                                 | <Box>`Intl.DateTimeFormat('en-US', { dateStyle: 'medium' }).format(date)`</Box>                                              |
+| Numeric locale-specific format | {Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)}                | When the user needs to compare across dates, such as in a column or list. Otherwise, use written format. Numeric format can differ depending on the region, so even if the date is translated by the UI based on the user's locale, numeric dates can still cause confusion for users. | <Box>`Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)`</Box>                |
+| ISO 8601 format                | {date.toISOString()}                                                                                              | Log files; trigger times.                                                                                                                                                                                                                                                              | <Box>`date.toISOString()`</Box>                                                                                              |
```

Comment:
Can we initialize date with a time so that this example presents something with a more realistic timestamp?

Reply:
Yes will update.